octave_pkg/DESCRIPTION

```diff
@@ -0,0 +1,14 @@
+Name: chebfun
+Version: 5.7.0+
+Date: 2025-08-09
+Author: The Chebfun Developers
+Maintainer: Colin B. Macdonald <cbm@m.fsf.org>
```

Comment:
@thomaskolman Didn't want to leave you out.  In fact there is a better metadata file that is more expressive that we'll create later, something like this:
https://github.com/gnu-octave/packages/blob/main/packages/symbolic.yaml